src/deadline/blender_submitter/addons/deadline_cloud_blender_submitter/open_deadline_cloud_dialog.py

```diff
@@ -52,13 +52,38 @@ def create_deadline_dialog(parent=None) -> SubmitJobToDeadlineDialog:
 
     # For the output path, first check for a value in the Scene settings
     if os.path.dirname(bpy.context.scene.render.filepath):
-        settings.output_path = os.path.dirname(bpy.context.scene.render.filepath)
+        settings.output_directory = os.path.dirname(bpy.context.scene.render.filepath)
     # If none, use the one in Preferences
     elif bpy.context.preferences.filepaths.render_output_directory:
-        settings.output_path = bpy.context.preferences.filepaths.render_output_directory
+        settings.output_directory = bpy.context.preferences.filepaths.render_output_directory
     # If neither of these are set, use the job bundle directory by default
     else:
-        settings.output_path = os.path.dirname(bpy.context.blend_data.filepath)
+        settings.output_directory = os.path.dirname(bpy.context.blend_data.filepath)
+
+    if bpy.path.basename(bpy.context.scene.render.filepath):
+        settings.output_file_prefix = bpy.path.basename(bpy.context.scene.render.filepath)
+
+    # When setting the output format, we can only access the extension from the Blender API.
+    # Here, we map them to formats available in the submitter GUI combobox.
+    file_formats = {
+        ".rbg": "IRIS",
+        ".png": "PNG",
+        ".jpg": "JPEG",
+        ".jp2": "JP2",
+        ".tga": "TGA",  # Can represent both TGA and RAWTGA - we leave it up to the user to specify
+        ".cin": "CINEON",
+        ".dpx": "DPX",
+        ".exr": "OPEN_EXR",  # Again, could be both OPEN_EXR and OPEN_EXR_MULTILAYER
+        ".hdr": "HDR",
+        ".tif": "TIFF",
+        ".webp": "WEBP",
+    }
+    try:
```

Comment:
nit: Could simplify this a little with `settings.output_format = file_formats.get(bpy.context.scene.render.file_extension, "PNG")`

Reply:
Didn't know that existed! Thank you for the pointer, will fix